defaults/main.yml

```diff
@@ -155,8 +155,8 @@ rke2_disable_cloud_controller: false
 # applicable only if rke2_disable_cloud_controller is true
 rke2_cloud_provider_name: "external"
 
-# Path to custom manifests deployed during the RKE2 installation
-# It is possible to use Jinja2 templating in the manifests
+# Path to folder with custom manifests deployed during the RKE2 installation
```

Comment:
I thin't it's better to keep it as a list of manifests. They can be in different folders or subfolders